docs/dev/clusterversion.md

```diff
@@ -52,7 +52,7 @@ metadata:
   namespace: openshift-cluster-network-operator
 $ cat <<EOF >version-patch.yaml
 > - op: add
->   path: /spec/overrides/-
+>   path: /spec/overrides
```

Comment:
This allows addition of overrides field with the value below, however, any addition of elements to the overrides overwrites rather than appends.  Unless I'm doing something wrong, there's a bug in json-patch/jsonpatch?